diff --git a/src/gasket_page_table.c b/src/gasket_page_table.c
index c9067cb..41dd598 100644
--- a/src/gasket_page_table.c
+++ b/src/gasket_page_table.c
@@ -54,7 +54,11 @@
 #include <linux/vmalloc.h>
 
 #if __has_include(<linux/dma-buf.h>)
-MODULE_IMPORT_NS(DMA_BUF);
+	#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 13, 0)
+	MODULE_IMPORT_NS(DMA_BUF);
+	#else
+	MODULE_IMPORT_NS("DMA_BUF");
+	#endif
 #endif
 
 #include "gasket_constants.h"
